tests/Adapter/Builder/LocalAdapterDefinitionBuilderTest.php

```diff
@@ -78,18 +78,18 @@ public function provideValidOptions()
      */
     public function testCreateDefinition($options)
     {
-        $this->assertSame(Local::class, $this->createBuilder()->createDefinition($options)->getClass());
+        $this->assertSame(LocalFilesystemAdapter::class, $this->createBuilder()->createDefinition($options)->getClass());
     }
 
     public function testOptionsBehavior()
     {
         $permissions = [
             'file' => [
-                'public' => 0755,
-                'private' => 0755,
+                'public' => 0666,
```

Comment:
This feels like a bug as the config didn't change

Reply:
Right, I was checking something during development and missed reverting this change.